plugins/transforms/selectvalues/pom.xml

```diff
@@ -29,4 +29,12 @@
     <packaging>jar</packaging>
     <name>Hop Plugins Transforms Select Values</name>
 
+    <dependencies>
```

Comment:
added this for writing assertions in a comfortable and readable way. Ideally we could introduce it in parent project. Will send a proposal.